book-3-bangazon/chapters/GR_REPORTS.md

```diff
@@ -0,0 +1,22 @@
+# Gamer Rater data reports
+The CEO of your startup is preparing the very first Annual Report to present to some of the backers who helped get the Gamer Rater platform off the ground. You have been asked to generate some data to include as part of the report.  
+
+Use your SQL skills to find the answers to the following questions. Create a `reports.sql` file and save it as part of your Gamer Rater repo.
```

Comment:
Please change this sentence to...

You are going to use your SQL skills, the `sqlite3` package in Python, and Django templates to create the following HTML reports .

Reply:
Ah, OK. Missed the memo on this being a Django template thing. I was thinking it was like the music history or Chinook exercises.

So, is this a new app in the Gamer Rater project?

Reply:
Is this one template/view in total, or a separate template for each report? Any additional guidance on setting this up that they need?

Reply:
Updated per comment. Happy to add more instrux if needed